ci/shared/scripts/deps/ldms.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -exo pipefail
+
+if test $# -lt 1
+then
+    echo "usage: ./$0 <ldms-version>"
+    exit 1
+fi
+
+version=$1
+
+echo "${version}"
+
+git clone https://github.com/ovis-hpc/ovis.git
+cd ovis
+git checkout OVIS-${version}
+
+./autogen.sh
+sed -i '19s|^prefix=.*|prefix=/opt/ldms|' ./packaging/make-all-top.sh
```

Comment:
We use `sed` here because `make-all-top.sh` has the installation directory hardcoded.